src/util/rlog/logger.rs

```diff
@@ -0,0 +1,60 @@
+use log::{debug, error, info};
+
+// Logger is a low-tech context logger. To add context call add_context().
+// To pass it around clone() it. If you can't do that then pass its
```

Comment:
Cloning this just to pass it to a new stack frame seems .. unfortunate. Can we pass by reference typically?